src/blosc2/CMakeLists.txt

```
@@ -18,13 +27,9 @@ else()
     set(BLOSC_INSTALL ON)
     add_subdirectory(c-blosc2)
     include_directories("${CMAKE_CURRENT_SOURCE_DIR}/c-blosc2/include")
```

```suggestion
    option(BLOSC_INSTALL "python-blosc2: Override" OFF)
    add_subdirectory(${PROJECT_SOURCE_DIR}/c-blosc2)
    include_directories("${PROJECT_SOURCE_DIR}/c-blosc2/include")
```

I've noticed your intent was to not package `blosc` with this [^1], in which case you should turn installation off since you are consuming the static library. Although there is nothing inherently wrong with adding the header and library files if you think other projects would want to depend on the C library instead.

The other 2 changes are related to the move of the files

[^1]: https://github.com/Blosc/python-blosc2/issues/128

---

Yeah, as said previously, I'd like to keep distributing the static C libraries inside the wheel.

And I have figured out how to avoid including c-blosc2 sources in latest commits.  I suppose what is left is a modernization of the C-Blosc2 CMake config files, but that should be for another day (and probably, another person more knowledgeable).

---

@LecrisUT, you have been of great help. Thanks again!

---

[Here](https://github.com/LecrisUT/CMake-Template) is my CMake template. I still need to tweak it in parts, but the main points of how structure it, make it readable, use modern CMake standards, and make it FetchContent compatible are there. Always looking for comments from people less experienced so that I can improve the documentation.

(I'll add `scikit-build-core` build in the next iteration, but without python bindings)